### ISSUE TITLE
Accessibility issues in tooltip

### ISSUE THREAD

**Issue by juuso-vaadin** [vaadin-component-factory/vcf-tooltip#2]
Did a quick a11y evaluation and came up with these issues.

Requirement | Result
-- | --
Opens on keyboard focus | OK
Can be opened / closed using keyboard | Can be configured in this way
Cursor can move across tooltip without dismissing the tooltip. | OK
Tooltip is dismissable with keyboard | Not OK. User needs to be able to dismiss the tooltip by pressing “Esc”.
Tooltip is dismissable with mouse | Not OK. User needs to be able to dismiss the tooltip by clicking a close button. (We should discuss this one)
Is announced by screen readers | Not OK. Should utilise aria-describedby to connect the element and tooltip.
Has aria-hidden when not visible | Not OK.

More info: https://sarahmhigley.com/writing/tooltips-in-wcag-21/#best-practices-summary
